securitas/security/ipa.py

```diff
@@ -21,6 +21,21 @@ def parse_group_management_error(data):
             raise ValidationError(failed)
 
 
+def parse_batch_error(data):
+    """
+    An extension of freeipa's function to handle batch requests.
+
+    TODO: send this upstream.
+    """
+    try:
+        error = data['error']
+    except KeyError:
+        return
+    # a normal respose has {"error": None} 
+    if error:
+        raise BadRequest(error)
```

Comment:
I think the `BadRequest` constructor accepts a `message` and a `code` as arguments, it would be better to make use of it with what FreeIPA returns.